### DISCUSSION TITLE
Search Engine Optimisation with pynecone

### DISCUSSION THREAD

**Discussion by NanoscaleSimulations** [reflex-dev/reflex#608] [Q&A]
Hey community and pynecone team

I am starting to learn web development and thanks to pynecone i can finally do it with python. However, from my perspective, SEO is a major part of building a website. What would be best practices for doing that or are there any pynecone components that are useful for that.

Kind regards

**Comment by Shahin-rmz** [comment]
SEO has two major parts.
1. technicasl
2. content.
Content is the major factor and relevant and good content plays almost 60% of the SEO.
About technical issues, i.e how fast the website loaded and other technical issues, I let Pynecone team to talk.
